Trading/Mode/daily_trading_mode/daily_trading.py

```diff
@@ -242,6 +255,25 @@ async def _get_quantity_ratio(self, currency):
         else:
             return 1
 
+    async def _get_max_amount_from_max_ratio(self, max_ratio, quantity, quote, default_ratio):
+        # reduce max amount when self.MAX_CURRENCY_RATIO is defined
+        if self.MAX_CURRENCY_RATIO is None or max_ratio == trading_constants.ONE:
+            return quantity * default_ratio
+        else:
+            max_amount_ratio = max_ratio - await self._get_ratio(quote)
```

Comment:
I think we could part with all these `await` because `get_holdings_ratio` (called by `_get_ratio`) doesn't need it